css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,19 @@
+button {
+  width: 150px;
+  height: 70px;
+}
+
+button:hover {
+  background-color: #000;
+  color: white;
+  font-size: large;
+}
+
+thead tr {
+  background-color: black;
+  font-weight: bold;
+}
+
+tr:nth-child(2n + 1) {
```

Comment:
another way to do this would be to do `tbody > tr:nth-child(odd)` and set the `thead` to:
```
color: white;
background-color: black;
```